src/SprykerEco/Zed/AmazonQuicksight/AmazonQuicksightConfig.php

```diff
@@ -86,15 +86,15 @@ public function getAwsAccountId(): string
 
     /**
      * Specification:
-     * - Returns the namespace for the Quicksight user registration.
+     * - Returns the name of the Quicksight namespace.
      *
      * @api
      *
      * @return string
      */
-    public function getQuicksightRegisterUserNamespace(): string
+    public function getQuicksightNamespace(): string
     {
-        return static::QUICKSIGHT_REGISTER_USER_NAMESPACE;
+        return $this->get(AmazonQuicksightConstants::AWS_QUICKSIGHT_NAMESPACE);
     }
```

Comment:
The method name must be getAwsQuicksightNamespace according to the constant name or constant name should be QUICKSIGHT_NAMESPACE